static/app/views/starfish/views/appStartup/screenSummary/index.tsx

```diff
@@ -84,48 +88,93 @@ function ScreenSummary() {
+                      },
+                      {
+                        type: 'count',
+                        dataKey: 'count',
+                        title: t('Count'),
+                      },
+                    ]}
+                    referrer="api.starfish.mobile-screen-totals"
+                  />
                 </Container>
+              </PageFiltersContainer>
+              <ErrorBoundary mini>
+                <AppStartWidgets additionalFilters={[`transaction:${transactionName}`]} />
+              </ErrorBoundary>
+              <EventSamplesContainer>
                 <ErrorBoundary mini>
-                  <AppStartWidgets
-                    additionalFilters={[`transaction:${transactionName}`]}
-                  />
+                  <div>
+                    <EventSamples
+                      cursorName={MobileCursors.RELEASE_1_EVENT_SAMPLE_TABLE}
+                      sortKey={MobileSortKeys.RELEASE_1_EVENT_SAMPLE_TABLE}
+                      release={primaryRelease}
+                      transaction={transactionName}
+                      showDeviceClassSelector
+                    />
+                  </div>
                 </ErrorBoundary>
-                <EventSamplesContainer>
-                  <ErrorBoundary mini>
-                    <div>
-                      <EventSamples
-                        cursorName={MobileCursors.RELEASE_1_EVENT_SAMPLE_TABLE}
-                        sortKey={MobileSortKeys.RELEASE_1_EVENT_SAMPLE_TABLE}
-                        release={primaryRelease}
-                        transaction={transactionName}
-                        showDeviceClassSelector
-                      />
-                    </div>
-                  </ErrorBoundary>
-                  <ErrorBoundary mini>
-                    <div>
-                      <EventSamples
-                        cursorName={MobileCursors.RELEASE_2_EVENT_SAMPLE_TABLE}
-                        sortKey={MobileSortKeys.RELEASE_2_EVENT_SAMPLE_TABLE}
-                        release={secondaryRelease}
-                        transaction={transactionName}
-                      />
-                    </div>
-                  </ErrorBoundary>
-                </EventSamplesContainer>
                 <ErrorBoundary mini>
-                  <SpanOperationTable
-                    transaction={transactionName}
-                    primaryRelease={primaryRelease}
-                    secondaryRelease={secondaryRelease}
-                  />
+                  <div>
+                    <EventSamples
+                      cursorName={MobileCursors.RELEASE_2_EVENT_SAMPLE_TABLE}
+                      sortKey={MobileSortKeys.RELEASE_2_EVENT_SAMPLE_TABLE}
+                      release={secondaryRelease}
+                      transaction={transactionName}
+                    />
+                  </div>
                 </ErrorBoundary>
-              </PageFiltersContainer>
```

Comment:
I realized I was adding components to this Container element so I moved the closing tag up